Mail/Views/Thread/SelectableTextView.swift

```diff
@@ -32,11 +36,15 @@ struct SelectableTextView: UIViewRepresentable {
         textView.textColor = UIColor(MailTextStyle.body.color)
         textView.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
 
-        textView.text = text
         return textView
     }
 
     func updateUIView(_ uiView: UITextView, context: Context) {
         uiView.text = text
+
+        DispatchQueue.main.async {
```

Comment:
Do we need to wait at least one run loop after setting the text property or would using @MainActor for the `updateUIView` func would suffice?